Topic 0/src/point2/Architect.java

```diff
@@ -0,0 +1,39 @@
+package point2;
+
+import java.util.ArrayList;
+
+
+public class Architect {
+
+	private Builder roombuilder;
+	private ArrayList<Room> rooms; //room list includes the architect to build the house
```

Comment:
[Medium] When you declare a variable, it is better to refer Interfaces or general classes when that it's possible. In this case for example,
```java
   List<Room> rooms = new ArrayList<Room>();
```
This will allow you to change the list structure if you needed, without any other changes in our code.